src/utils/deleteContracts.ts

```diff
@@ -24,22 +26,25 @@ export default async function deleteContracts(
     "tfkvstore" as any
   );
 
-  // remove named vms
+  // remove deployment in namespace `type`
   console.log({ grid });
   const obj = await grid.connect().then(() => {
     return grid[key].delete({ name });
   });
 
-  if (obj["deleted"].length) {
-    return obj;
-  }
+  // if (obj["deleted"].length) {
+  //   return obj;
+  // }
```

Comment:
why this part is commented? I think it's used for preventing deleting a machine with the same name from the default namespace.

Reply:
if it is enabled. only the VM in the project namespace deleted we commented it so we can skip to the gateway deletion. going to refactor that

Reply:
but if your deployment is called `test` in peertube's namespace and also have a machine is called `test` in the default namespace, then tried to delete the one on peertube's namespace, it will also delete the other machine